scIB/metrics/ari.py

```diff
@@ -1,10 +1,12 @@
+import numpy as np
 import pandas as pd
+import scipy.special
 from sklearn.metrics.cluster import adjusted_rand_score
 
 from scIB.utils import checkAdata, checkBatch
 
 
-def ari(adata, group1, group2):
+def ari(adata, group1, group2, implementation='sklearn'):
```

Comment:
why do we default to an implementation that potentially will not work? I would probably do the opposite here.